drain-service/drain3/drain.py

```diff
@@ -19,6 +19,9 @@ def __init__(self, log_template_tokens: list, cluster_id: int, is_pretrained = F
         self.pretrained_anomaly_level = pretrained_anomaly_level
         self.cache = dict()
         self.size = 1
+        self.max_frequency_original_log = ""
+        self.max_frequency_masked_log = ""
+        self.max_frequency_count = -1
```

Comment:
this variable represents the `max frequency of ALL TIME`.
however users might be more interested in `max frequent template in the last X hours`?

Reply:
Perhaps we can introduce that change down the road. That would require additional changes to the code base. I can add a card for that going forward.